src/schema/budget-line.ts

```diff
@@ -0,0 +1,22 @@
+import { pgTable, primaryKey, text } from "drizzle-orm/pg-core";
+import { z } from "zod";
+
+export const budgetLine = pgTable(
+  "budget_line",
+  {
+    code: text("code"),
```

Comment:
We'll also want the foreign key to the agency budget code. Let me know if this works in your editor

```js
    code: text("code").references(() => agencyBudget.code),
```

Reply:
Looks ok to me locally